Clinical-Genomics/trailblazer
# Continues when mip data is missing

This PR makes trailblazer get data from mip output without failing as soon as the structure is different

How to test:
1. install on rasta stage
1. us
1. run `cg upload delivery-report -p [custid] [familyid]` on an somewhat older case that fails in production with `KeyError`

Expected output:
A delivery report
